gui/slick/views/config_providers.mako

```diff
@@ -168,6 +168,18 @@ $('#config-components').tabs();
                         </div>
                         % endif
 
+                        % if hasattr(curNewznabProvider, 'enable_manualsearch'):
+                        <div class="field-pair">
+                            <label for="${curNewznabProvider.get_id()}_enable_manualsearch">
+                                <span class="component-title">Enable provider for manual searches</span>
+                                <span class="component-desc">
+                                    <input type="checkbox" name="${curNewznabProvider.get_id()}_enable_manualsearch" id="${curNewznabProvider.get_id()}_enable_manualsearch" ${('', 'checked="checked"')[bool(curNewznabProvider.enable_manualsearch)]}/>
```

Comment:
These lines are absurdly long. And some of the method calls are being unneccessarily repeatedly (specifically `.get_id()`).  
If you add a single `<% cur_id = curNewznabProvider.get_id() %>` right after the for loop.  Then all the `curNewznabProvider.get_id()` can be replaced by `cur_id`.
This line then gets shortened to
`<input type="checkbox" name="${cur_id}_enable_manualsearch" id="${cur_id}_enable_manualsearch" ${('', 'checked="checked"')[bool(curNewznabProvider.enable_manualsearch)]}/>`

The second have can also be rewriiten (and more clearly) like this
`{'checked="checked"' if curNewznabProvider.enable_manualsearch else ''}`

So the final line would be
`<input type="checkbox" name="${cur_id}_enable_manualsearch" id="${cur_id}_enable_manualsearch" ${'checked="checked"' if curNewznabProvider.enable_manualsearch else ''}/>`